src/features/common/AuthFireRoute.js

```diff
@@ -3,8 +3,12 @@ import PropTypes from "prop-types";
 import React from "react";
 import { useAuth } from "../../contexts/AuthContext";
 
-//AuthFirebaseRouteはログインの有無によって遷移ページを変える関数
-//ログインしていない場合はリダイレクトされる
+/**
+ * AuthFirebaseRouteはログインの有無によって遷移ページを変える関数
+ * ログインしていない場合はリダイレクトされる
+ * @params {component: 遷移する先のコンポーネント名, path: そのコンポーネントまでのパス}
+ * @return {Component}} ログインページのリダイレクトか, pathへ遷移するComponent
```

Comment:
```suggestion
 * @param {Component} component 遷移する先のコンポーネント名
 * @param {string} path そのコンポーネントまでのパス
 * @return {Component} ログインページのリダイレクトか, pathへ遷移するComponent
```

nits.
githubのハイライトも効くので👆のスタイルに統一しましょう．